command/report/report.go

```diff
@@ -127,7 +127,7 @@ func (opts *ReportOptions) Run() int {
 		return keys
 	}
 
-	if !supportedKeys[reportCommandKey] {
+	if reportCommandAnalyzerShortcode == "test-coverage" && !supportedKeys[reportCommandKey] {
```

Comment:
Does the logic following this `if` block make sense if the analyzer is not `test-coverage`?

Reply:
The logic in the if block makes sense only for test-coverage analyzer. `key` was designed to be an optional field, and only test coverage analyzer accepts artifacts with multiple keys.

Everything else after this if block makes sense for all analyzers.